File: 455.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        if len(g) == 0 or len(s) == 0:
            return 0
        
        g.sort()
        s.sort()
        index = 0
        result = 0

        for num in s:
            if index < len(g) and num >= g[index]:
                result += 1
                index += 1
        
        return result

File: 455.java
class Solution {
    public int findContentChildren(int[] g, int[] s) {
        if(s.length == 0 || g.length == 0) return 0;
        
        Arrays.sort(g);
        Arrays.sort(s);
        int result = 0;
        int index = 0;

        for(int i = 0; i < s.length; i++) {
            if(index < g.length && s[i] >= g[index]){
                result++;
                index++;
            } 
        }
        return result;
    }
}
